apps/omg_rpc/test/omg_rpc/web/plugs/health_test.exs

```diff
@@ -0,0 +1,99 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.RPC.Plugs.HealthTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: true
+
+  alias OMG.RPC.Web.TestHelper
+  @alarm_1 {:boot_in_progress, %{node: Node.self(), reporter: __MODULE__}}
+  @alarm_2 {:ethereum_client_connection, %{node: Node.self(), reporter: __MODULE__}}
+  describe "testing for boot_in_progress alarm" do
+    @tag fixtures: [:phoenix_sandbox]
+    test "if block.get endpoint rejects request because alarms are raised" do
+      :ok = :alarm_handler.set_alarm(@alarm_1)
+
+      assert %{
+               "data" => %{
+                 "code" => "operation:service_unavailable",
+                 "description" => "The server is not ready to handle the request.",
+                 "object" => "error"
+               },
+               "success" => false,
+               "version" => "1.0"
+             } = TestHelper.rpc_call(:post, "/block.get", %{})
+
+      :ok = :alarm_handler.clear_alarm(@alarm_1)
+    end
+
+    @tag fixtures: [:phoenix_sandbox]
+    test "if block.get endpoint rejects the request because of bad params when alarm is cleared" do
```

Comment:
this and the other `bad_request` tests seem either redundant or misplaced, at 1st glance.

If we want these to be sanity checks, or checks of some behavior following `clear_alarm`, I suggest the following:
 - strip down the assertion to something more minimal, e.g. only match that the `"code"` **isn't** `service_unavailable` anymore, after alarm had cleared
 - organize these assertions in more natural scenarios like e.g.:
    1. set alarm 1
    2. check that `service_unavailable`, checking correct format of the whole response here
    3. clear alarm 1
    4. check that `not service_unavailable`, ignoring the rest of the response